CONTRIBUTING.md

```diff
@@ -28,26 +29,33 @@ To contribute a patch, the workflow is as follows:
   1. Create topic branch
   1. Commit patches
 
-The project coding conventions in the [developer notes](doc/developer-notes.md)
-must be adhered to.
+The project coding conventions in the [developer notes] must
+be adhered to.
 
-In general [commits should be atomic](https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention)
-and diffs should be easy to read. For this reason do not mix any formatting
-fixes or code moves with actual code changes.
+[developer notes]: doc/developer-notes.md
+
+In general [commits should be atomic][c] and diffs should be easy
+to read. For this reason do not mix any formatting fixes or code moves
+with actual code changes.
+
+[c]: https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention
 
 Commit messages should be verbose by default consisting of a short subject line
 (50 chars max), a blank line and detailed explanatory text as separate
 paragraph(s), unless the title alone is self-explanatory (like "Corrected typo
 in init.cpp") in which case a single title line is sufficient. Commit messages should be
 helpful to people reading your code in the future, so explain the reasoning for
-your decisions. Further explanation [here](http://chris.beams.io/posts/git-commit/).
+your decisions. Further explanation [here].
+
+[here]: http://chris.beams.io/posts/git-commit/
```

Comment:
If we're touching this please make this link https

Reply:
Thanks for suggestion. As the https link works, there it is now.